projects/core/src/cart/store/selectors/cart.selector.ts

```diff
@@ -69,6 +69,11 @@ export const getLoaded: MemoizedSelector<any, boolean> = createSelector(
     !loaderValueSelector(state).refresh
 );
 
+export const getLoading: MemoizedSelector<any, boolean> = createSelector(
```

Comment:
FYI function `getLoading` is exposed from the core lib. Can you precise it to `getCartLoading`? 
Btw. if not a big effort, could you also precise neighbour functions in this file?
```
getRefresh
getEntriesMap
getEntrySelectorFactory
getEntries
```